service/src/main/java/tds/assessment/services/impl/AssessmentAssembler.java

```diff
@@ -0,0 +1,69 @@
+package tds.assessment.services.impl;
+
+import com.google.common.collect.ArrayListMultimap;
+import com.google.common.collect.Multimap;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import tds.assessment.Assessment;
+import tds.assessment.Form;
+import tds.assessment.Item;
+import tds.assessment.ItemConstraint;
+import tds.assessment.ItemProperty;
+import tds.assessment.Segment;
+import tds.assessment.Strand;
+
+/**
+ * A helper class for assembling the {@link tds.assessment.Assessment}
+ */
+class AssessmentAssembler {
+
+    /**
+     * Maps each {@link tds.assessment.ItemProperty} to its respective {@link tds.assessment.Item}
+     *
+     * @param items
+     * @param itemProperties
+     */
+    static void assignPropertiesToItems(List<Item> items, List<ItemProperty> itemProperties) {
+        Multimap<String, ItemProperty> itemIdToPropertyMap = ArrayListMultimap.create();
```

Comment:
Why use a multimap rather than just have a `Map<String, List<ItemProperty>>`?  It doesn't look like we're using any of the specific features that a MultiMap would by us.